protocol/contracts/lease/src/position/spec/mod.rs

```diff
@@ -76,6 +79,26 @@ where
         }
     }
 
+    /// Calculate the borrow amount.
+    /// Return 'error::ContractError::InsufficientTrasactionAmount' when either the downpayment
+    /// or the borrow amount is with amount less than the minimum trasaction amount.
+    /// Return 'error::ContractError::InsufficientAssetAmount' when the lease (downpayment + borrow)
+    /// is with amount less than the minimum asset amount.
+    pub fn calc_borrow_amount(
+        &self,
+        downpayment: Coin<Lpn>,
+        may_max_ltd: Option<Percent>,
+    ) -> ContractResult<Coin<Lpn>> {
+        let amount_in_lpn: Price<Lpn, Lpn> = Price::identity();
```

Comment:
1. let's not specify the type when the type inference manages to do it
2. the variable name is not correct, that is not an amount